### PR TITLE
PZ-2667 Document credit card mandate export procedure for competing PSPs

### PR DESCRIPTION
**Pull request by willemstuursma**

### CODE REVIEW

File: source/payments/importing-mandates.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
+Importing credit card mandates from Mollie
+==========================================
+
+.. note:: This document is aimed at competing payment service providers who wish to import existing
+          credit card mandates from Mollie for a merchant.
+
+In the event that a merchant wishes to switch to a different payment service provider, the merchant
+can export a list of his / her credit card mandates from the Mollie Dashboard.
+
+You must provide the merchant with the export format you would like to receive and a public `OpenPGP <https://www.openpgp.org/>`_
```
Comment (pabloroman) [inline source/payments/importing-mandates.rst:10]: What are the different export formats we provide?

File: source/payments/importing-mandates.rst [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+key the export should be encrypted to.
+
+The merchant must contact Mollie to enable the functionality to export the mandates.
+
+Then, the merchant can upload your public key in the Mollie Dashboard and export the mandates from
+there. Additionally, a public key of Mollie must be downloaded.
```
Comment (pabloroman) [inline source/payments/importing-mandates.rst:16]: Where can this public key be found? also in the Dashboard?

Comment (willemstuursma) [reply source/payments/importing-mandates.rst:16]: Yes.

File: source/payments/importing-mandates.rst (hunk 1 above)
Comment (willemstuursma) [reply source/payments/importing-mandates.rst:10]: I'll clarify. I don't want to name any competitors.

File: source/payments/importing-mandates.rst [hunk 3]
```diff
@@ -0,0 +1,89 @@
+Importing credit card mandates from Mollie
```
Comment (sandervanhooft) [inline source/payments/importing-mandates.rst:1]: Shouldn't the title be "Exporting" instead of "Importing"?

Comment (sandervanhooft) [reply source/payments/importing-mandates.rst:1]: If so, the filename should be `exporting-mandates.rst`

Comment (willemstuursma) [reply source/payments/importing-mandates.rst:1]: Hi Sander. This is for employees of competing PSPs that wish to import mandates from Mollie. So from their pov it is importing as it mostly deals with the decryption and signature validation.

We will provide public documentation on exporting mandates from competing PSPs later.

**Comment by sandervanhooft** [comment]
Can this also be used for moving mandates between Mollie accounts? Could be valuable for profile carve-outs (I.e. a profile that becomes a separate business entity)

**Comment by willemstuursma** [comment]
Hi @sandervanhooft, that is an interesting idea. I'll come back to you on that.

**Pull request merged**